block/manager.go

```diff
@@ -9,17 +9,18 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/dymensionxyz/dymint/utilevent"
```

Comment:
pls change to `dymint/utils/events` if u want separate package for event utils

Reply:
done 👍 
Yes defo want different packages for different kinds of util
a global util pkg is terrible 😄